src/Peering/Peering/Peering/GetAzurePeeringCommand.cs

```diff
@@ -69,6 +65,17 @@ public class GetAzurePeeringCommand : PeeringBaseCmdlet
         [PSArgumentCompleter(Constants.Direct, Constants.Partner, Constants.Exchange)]
         public string Kind { get; set; }
 
+        /// <summary>
+        /// The resource  id
+        /// </summary>
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            HelpMessage = Constants.ResourceIdHelp,
+            ParameterSetName = Constants.ParameterSetNameByResourceId)]
```

Comment:
@rileymckenna make sure that this parameter has `ValueFromPipelineByPropertyName = true`